### ISSUE TITLE
update dataone timeline on website

### ISSUE THREAD

**Issue by mbjones** [DataONEorg/community-engagement-outreach#39]
The current web site has a timeline that ends in 2019 when we switch leadership. Let's add some additional info about progress over the last four years.

https://www.dataone.org/about/

**Comment by angiegarciaa** [comment]
Are there any major events you would like to be highlighted?

**Comment by mbjones** [comment]
Yeah, probably our new member growth, support for schema.org as a membership avenue, and the development and rollout of our sustainability service model for Hosted Repositories and DataONE Plus. Let's chat about that with the rest of the team on our next call.
